### ISSUE TITLE
[1.10.2][Request/Suggestion/Bug] Config Option & Drill Missing Ores

### ISSUE THREAD

**Issue by Gebcrafter** [BluSunrize/ImmersiveEngineering#1635]
I have been using the mining drill and i would enjoy a config option for the durability of the drill head, it seems to go down very fast, with no way to repair it evident to me. Also, can we have a secondary speed increase? One that requires more lubrication, etc., because i feel that the drill after the first speed upgrade could use a little more speed. BUG: when im mining a straight tunnel with the 3x3 head, it only mines the soft blocks (stone, dirt, marble, etc.) and leaves the ores behind. This may also have to do with the fact that it only mines one type of block, in its 3x3. If i were to mine a 3x3 wall made up of dirt & cobble, it would only mine all of whatever block is in the center (so, if dirt was in the center it would break all dirt around it in a 3x3, not cobble).

**Comment by mindforger** [comment]
> BUG: when im mining a straight tunnel with the 3x3 head, it only mines the soft blocks (stone, dirt, marble, etc.) and leaves the ores behind

this is no bug it is intended, the drill always mines all blocks with roughly the same hardness like the block you are working on, so when you work on an ore block it will mine all blocks around it with the same or less hardness

the only thing i can think of would be some sort of upgrade

**Comment by Doragoon01** [comment]
I'm sorry for what feels like piling on...  but...
Is it intended for the mining drill to be slower than a vanilla diamond pick with efficiency 5 and a haste 2 beacon?  And it's durability seems to go down faster than said diamond pick with unbreaking 3.  And while it still drains your hunger at the same rate, it also is using diesel fuel.  I like the idea of the mining drill,  but i'm having trouble justifying it.

**Comment by mindforger** [comment]
I am not challenging the other details, I was just pointing out that digging on stone is leaving behind "harder" materials is working as intended :) sorry for beeing unspecific

**Comment by Gebcrafter** [comment]
@mindforger so you would agree that the speed needs to be buffed? I do, it takes lots of steel & other resources to not only make it, but get its fuel. I made a hammer from tinkers and it worked far better, there needs to be something about the drill that would make me be inclined to use it over a hammer.

**Comment by mindforger** [comment]
to be honest, the tinker tools are kind of overpowered but having the option to tune the drill would be kind of nice to be able to balance everythign out in a modpack for example

buffing it in general is not the best solution as it is Blu's decision and he has reasons i guess :)

still, another upgrade for the drill that allows for 3x3 operation independant of hardness would be nice in some way ... maybe something like an "impact upgrade", let you work through the ground in a 3x3 without caring for material hardness, but stone gravel and dirt will be pulverized into fine dust and do not drop anymore. Also increases mining speed and fuel consumption :)

**Comment by BluSunrize** [comment]
There is an API, and I believe Glassmaker had plans to use it back in the day. However, because of life and general business, said mod is currently on ice.

**Comment by Gebcrafter** [comment]
@BluSunrize i would most definitely appreciate that mod. The drill would be so much better, and it would allow for upgrades for late-ish game

**Comment by BluSunrize** [comment]
I think the Drill is reasonably balanced at its current point. I've done comparissons to a common TCon Hammer setup (Cobalt head, Obsidian and Ardit plates, Blue Slime handle, 150 redstone)
The hammer is barely a second faster on a length of 32 blocks.
The hammer does have 'Duritae', which roughly halves the damage done to it I believe, but the drill head has 8000 Durability as opposed to the hammer's 2060.
The fuel is renewable.

Sure, TCon hammers are easier to make! Of course, the IE one is complex, requires more crafting, not as customizable and stuff. But they also don't really aspire to be .-.
The IE drills are supposed to give a solid feel, a bit of chunky machinery in your hands. Maybe not as diverse and colourful as TCon (if you ignore the awesome shader system ;3 ) but more so designed with IE's style in mind.

tl;dr? I don't think they need a stat buff right now.
Also, drill heads are repaired with ingots in the anvil ^^

As for the thing with hard and soft blocks, that is intentional and there is not much to be done about it.
The point to it is, that you will only dig stuff in parallel that is weaker or equally as strong, in order to prevent people from cheesing obsidian by digging cobblestone next to it.

**Comment by Gebcrafter** [comment]
@BluSunrize That's definitely fair, but the hardness thing is pretty annoying and there are plenty of other ways to cheese obsidian. Also another Bug i found; the drill head has -6 durability after it being at 3, then me mining a 3x3. (Which may be the reason why i couldn't/cant repair it in anvil).
